lib/update/cluster/engine_test.go

```diff
@@ -232,7 +234,14 @@ func (p *testPhase2) Rollback(context.Context) error {
 }
 
 func (r *testReconciler) ReconcilePlan(ctx context.Context, plan storage.OperationPlan) (*storage.OperationPlan, error) {
+	changes, err := r.backend.GetOperationPlanChangelog(plan.ClusterName, plan.OperationID)
```

Comment:
As the original plan is never modified, the reconciler needs to actually apply changes even in the test.